.github/workflows/publish_world_flags.yaml

```
@@ -0,0 +1,16 @@
+name: publish_world_flags
+
+on:
```

yamllint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)